lib/styles/settings/colors/_raw.json

```diff
@@ -1 +1 @@
e":"#DD2A3A"},{"id":"_raw.scss_red-600","label":"red-600","value":"#CA1E2E"},{"id":"_raw.scss_red-700","label":"red-700","value":"#B41E2D"},{"id":"_raw.scss_red-800","label":"red-800","value":"#9F1D2C"},{"id":"_raw.scss_red-900","label":"red-900","value":"#831C2B"}],"blue":[{"id":"_raw.scss_blue-50","label":"blue-50","value":"#E6F4FA"},{"id":"_raw.scss_blue-100","label":"blue-100","value":"#CDE8F5"},{"id":"_raw.scss_blue-200","label":"blue-200","value":"#9FD4EC"},{"id":"_raw.scss_blue-300","label":"blue-300","value":"#62ADD5"},{"id":"_raw.scss_blue-400","label":"blue-400","value":"#308BBF"},{"id":"_raw.scss_blue-500","label":"blue-500","value":"#1779B5"},{"id":"_raw.scss_blue-600","label":"blue-600","value":"#0C6CA8"},{"id":"_raw.scss_blue-700","label":"blue-700","value":"#0C6095"},{"id":"_raw.scss_blue-800","label":"blue-800","value":"#0C5685"},{"id":"_raw.scss_blue-900","label":"blue-900","value":"#0C4870"}],"violet":[{"id":"_raw.scss_violet-50","label":"violet-50","value":"#F4F1F8"},{"id":"_raw.scss_violet-100","label":"violet-100","value":"#E6DEEE"},{"id":"_raw.scss_violet-200","label":"violet-200","value":"#CEBFDE"},{"id":"_raw.scss_violet-300","label":"violet-300","value":"#AF97C8"},{"id":"_raw.scss_violet-400","label":"violet-400","value":"#9271B5"},{"id":"_raw.scss_violet-500","label":"violet-500","value":"#7E57A7"},{"id":"_raw.scss_violet-600","label":"violet-600","value":"#72499B"},{"id":"_raw.scss_violet-700","label":"violet-700","value":"#643D8C"},{"id":"_raw.scss_violet-800","label":"violet-800","value":"#56367B"},{"id":"_raw.scss_violet-900","label":"violet-900","value":"#472F6B"}],"indigo":[{"id":"_raw.scss_indigo-50","label":"indigo-50","value":"#F0F1FD"},{"id":"_raw.scss_indigo-100","label":"indigo-100","value":"#E4E5FB"},{"id":"_raw.scss_indigo-200","label":"indigo-200","value":"#BFC2F5"},{"id":"_raw.scss_indigo-300","label":"indigo-300","value":"#989DEE"},{"id":"_raw.scss_indigo-400","label":"indigo-400","value":"#7279E6"},{"id":"_raw.scss_indigo-500","label":"indigo-500","value":"#666ED0"},{"id":"_raw.scss_indigo-600","label":"indigo-600","value":"#5D64BE"},{"id":"_raw.scss_indigo-700","label":"indigo-700","value":"#5058A6"},{"id":"_raw.scss_indigo-800","label":"indigo-800","value":"#454D90"},{"id":"_raw.scss_indigo-900","label":"indigo-900","value":"#384179"}],"theme":[{"id":"_raw.scss_theme-50","label":"theme-50","value":"#E8F6F6"},{"id":"_raw.scss_theme-100","label":"theme-100","value":"#D5EEEF"},{"id":"_raw.scss_theme-200","label":"theme-200","value":"#A6DCDE"},{"id":"_raw.scss_theme-300","label":"theme-300","value":"#5BBDC1"},{"id":"_raw.scss_theme-400","label":"theme-400","value":"#0D9EA4"},{"id":"_raw.scss_theme-500","label":"theme-500","value":"#078F96"},{"id":"_raw.scss_theme-600","label":"theme-600","value":"#088289"},{"id":"_raw.scss_theme-700","label":"theme-700","value":"#096C76"},{"id":"_raw.scss_theme-800","label":"theme-800","value":"#095D68"},{"id":"_raw.scss_theme-900","label":"theme-900","value":"#0A4C58"}]}
```

Comment:
How did you update token files? Manually? I don't see any changes to parsing scripts 🤔

Reply:
The script is NSFW, but I had put it in the LMS and whole instruction how I went through it to have it imported:
* https://bethink.atlassian.net/browse/LMS-2561?focusedCommentId=158915